### ISSUE TITLE
Better Documentation for Fine-grained Personal Access Tokens

### ISSUE THREAD

**Issue by jslay88** [actions/actions-runner-controller#2361]
### What would you like added?
GitHub now supports fine-grained PAT. A guide that covers how to configure a token with LPA (least privileged access) for common scenarios would be a helpful guide. 

*A clear and concise description of what you want to happen.*
Documentation that leverages the new fine-grained PAT, and provides sets of permissions one may use to create a fine-grained PAT for common scenarios in which one would use ARC. 

### Why is this needed?
Having documentation that tells users to use PAT that provide large scale permissions is not the best security approach.  It is a common practice to generate tokens with LPA permissions.

*A clear and concise description of any alternative solutions or features you've considered.*
I am manually building my own sets of permissions at the moment for the various scenarios we use ARC in.

### Additional context

*Add any other context or screenshots about the feature request here.*
Fine-grained PAT now allows tokens for specific repositories, with more fine grained permission controls.
![image](https://user-images.githubusercontent.com/6158929/222873858-8f5511bd-7c48-4e16-9264-327941eafc12.png)

**Comment by chusband** [comment]
Agreed.  Fine-grained PAT are best security practice.  Please provide guidance

**Comment by harrisonfang-helm** [comment]
I got a repo-level runner up and running with a fine-grained PAT. Not sure if this is the best practice, but I found that repository administration read/write was necessary and sufficient.
